man/systemd.netdev.xml

```diff
@@ -527,9 +527,9 @@
 
     <variablelist class='network-directives'>
       <varlistentry>
-        <term><varname>Id=</varname></term>
+        <term><varname>VNI=</varname></term>
         <listitem>
-          <para>The VXLAN ID to use.</para>
+          <para>The VXLAN VNI Network Identifier (or VXLAN Segment ID). Ranges [0-16777215].</para>
```

Comment:
... so "VXLAN VNI Network Identifier" expands to "VXLAN VXLAN Network Identifier Network Identifier". Please just shorten this to "VXLAN Network Identifier" here.

Reply:
Yes updated . thanks.